xds/internal/balancer/ringhash/ringhash.go

```diff
@@ -94,6 +96,18 @@ type ringhashBalancer struct {
 	ring *ring
 }
 
+// hashKey returns the hash key to use for an endpoint. Per gRFC A61, each entry
+// in the ring is an endpoint, positioned based on the hash of the endpoint's
+// first address, unless the endpoint has an explicit hash key as described in
+// gRFC A76.
+func hashKey(endpoint resolver.Endpoint) string {
+	hk := ringhash.GetEndpointHashKey(endpoint)
+	if hk != "" {
+		return hk
+	}
```

Comment:
Nit: combine the assignment and the conditional into a single line.

Reply:
Done.